web/play.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Web frontend</title>
+    <script src="https://pixijs.download/release/pixi.js"></script>
+
+    <script src="script/display.js"></script>
+    <script src="script/fetch.js"></script>
+
+    <script>
+
+      function startLoop() {
+        const button = document.getElementById('restart-button');
+        button.style.display = 'none';
+
+        let displayWidth = 600;
+        let displayHeight = 300;
+        let out = setupGraphics(button, displayWidth, displayHeight);
+        let appView = out[0];
+        let graphics = out[1];
+        doFetch(button, appView, graphics, displayWidth, displayHeight);
+      }
+
+      window.onload=()=>{
+        startLoop();
+      }
+    </script>
+  </head>
+  <body>
+    <button id="restart-button" onclick="startLoop();" style="display: block;">Restart</button>
```

Comment:
When I run the web frontend I just see an image displaying a view of the world; I don't see any kind of "Restart" button.  I'm not sure I understand whether that is the expected behavior.

Reply:
Yeah, the restart button will (or is supposed to, anyway) appear only if an error is encountered.